### ISSUE TITLE
Question on UCR Datasets

### ISSUE THREAD

**Issue by Wwwwei** [time-series-machine-learning/tsml-java#541]
Hi, I am trying to follow some _Shapelets_ work in time series classification, such as 

- [Learning time-series shapelets](https://dl.acm.org/doi/abs/10.1145/2623330.2623613?casa_token=NESErENbRecAAAAA:E6So99Q5pKDgHqGQL9oiG12qvPB1KEsL0xncGOK5qUjz1fHGevCtuiTJXPeiznTzdRdMPnmN2gq1qw)
- [Time series shapelets: a new primitive for data mining](https://dl.acm.org/doi/abs/10.1145/1557019.1557122?casa_token=WUBHJxQosUYAAAAA%3A876JnI4A3EN36a36p8vUrxhgX0RqJJXLi9u9cD0qTc6aRwdDGfQYv2TSEHBrZ_scs0ZOiFkcP9837g)
- [Adversarial Dynamic Shapelet Network](https://ojs.aaai.org/index.php/AAAI/article/view/5948)

I notice that following datasets are widely used in these work:

- DP_Little
- DP_Middle
- DP_Thumb
- MP_Little
- MP_Middle
- PP_Little
- PP_Middle
- PP_Thumb

However, I did not find them in http://www.timeseriesclassification.com/ (although the most authors state that the datasets came from UCR archive). I would like to know whether these datasets have been renamed or where I could get them. 

Any one help me? Thank you for your assistance.

**Comment by TonyBagnall** [comment]
hi, we used these bone datasets for our shapelet paper https://scholar.google.com/citations?view_op=view_citation&hl=en&user=gxJ2GYQAAAAJ&citation_for_view=gxJ2GYQAAAAJ:M3NEmzRMIkIC but that was before we helped restructure the UCR archive. We dropped these datasets which were I think histogram intensities in favour of the outlines e.g. http://timeseriesclassification.com/description.php?Dataset=ProximalPhalanxOutlineCorrect. I dont think I still have them, I would instead use the current archive datasets
